uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-comments-tab.component.spec.js

```diff
@@ -85,10 +86,10 @@ define([
 
       it('adds comment with proper values', function () {
         expect(controller.request.comments[0]).toEqual({
-          contact_id: staffId,
+          contact_id: contactId,
           created_at: jasmine.any(String),
```

Comment:
It is used to show the comment immediately in the UI without saving. If you remove contact_id - it will render an empty text span.